lib/KettleMiddleware.js

```diff
@@ -173,10 +173,11 @@ fluid.defaults("kettle.middleware.CORS", {
 
 /**
  * A middleware responsible for enabling CORS within the kettle server.
- * @param {Object} request a request object.
- * @param {String|Array} allowMethods methods that are enabled with CORS.
- * @param {String|Array} origin domains that are allowed.
- * @param {String} credentials response exposure flag.
+ * @param {Component} request - a kettle.request component
+ * @param {String|String[]} allowMethods - methods that are enabled with CORS.
+ * @param {String|String[]} origin - domains that are allowed.
```

Comment:
If it didn't complain, it's also valid, but in previous work together we have favoured the `Array<String>` syntax.

Reply:
Also, if you decide to capitalise sentences, these need to be updated.